include/swift/Sema/SyntacticElementTarget.h

```diff
@@ -457,6 +464,11 @@ class SyntacticElementTarget {
     return cast<ExprPattern>(expression.pattern);
   }
 
+  ReturnStmt *getReturnStmt() const {
```

Comment:
Maybe `getParentReturnStmt()` instead to indicate that we are returning contextual node that wraps the expression?